tasks/main.yml

```diff
@@ -14,6 +14,8 @@
 
 - name: Handle system subscription
   include_tasks: subscription-manager.yml
+  tags:
+  - subscription-manager
```

Comment:
uhm, what is it for?

Reply:
Should be off, I use it to test Insights in a machine